Softeq.XToolkit.Common.Droid/Converters/VisibilityConverter.cs

```diff
@@ -0,0 +1,28 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Android.Views;
+using Softeq.XToolkit.Common.Converters;
+
+namespace Softeq.XToolkit.Common.Droid.Converters
+{
+    /// <summary>
+    ///     Converts <see cref="T:System.Boolean"/> to Android specific <see cref="T:Android.Views.ViewStates"/> and back.
+    /// </summary>
+    public class VisibilityConverter : IConverter<ViewStates, bool>
+    {
+        public static IConverter<ViewStates, bool> Instance { get; } = new VisibilityConverter();
+
+        /// <inheritdoc />
+        public ViewStates ConvertValue(bool value, object? parameter = null, string? language = null)
+        {
+            return value ? ViewStates.Visible : ViewStates.Invisible;
```

Comment:
`GoneConverter` can be changed to use `Invisible` or `Gone` option (via property or constructor).

Reply:
Continued in issue below #397